lms/djangoapps/support/static/support/jsx/logged_in_user.jsx

```diff
@@ -75,26 +76,29 @@ function LoggedInUser({ userInformation, setErrorState, zendeskApiHost, submitFo
     <div className="row">
       <div className="col-sm-12">
         <div className="form-group">
-          {courseElement}
+          {subjectElement}
```

Comment:
Make sure to check if there is any other place courseElement is used and if it should be updated as well to keep it consistent

Reply:
courseElement was just moved to the bottom, nothing about it was changed. I just moved its location in form.